controllers/platformnetwork_controller.go

```diff
@@ -385,7 +391,6 @@ func (r *PlatformNetworkReconciler) RestorePlatformNetworkStatus(instance *starl
 			instance.Status.Reconciled = true
 			instance.Status.ObservedGeneration = instance.ObjectMeta.Generation
 			instance.Status.DeploymentScope = "bootstrap"
-			instance.Status.StrategyRequired = "not_required"
```

Comment:
Why to remove the not_required here?

Reply:
The "StrategyRequired" field is not part of PlatformNetworkStatus anymore.
It is also not part of AddressPoolStatus.
The reason is VIM checks "strategy_required" of Host resource instance only to propagate VIM strategies (ie. lock/unlock) and the field seemed to serve no purpose on PlatformNetwork / AddressPool resources.

Reply:
acked